manifests/base/tasks/onboarding.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: task
+metadata:
+  name: run
+spec:
+  params:
+    - name: PAYLOAD
+      type: string
+      default: ""
+  steps:
+    - name: clone-repo
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        echo "Cloning repo"
+        git clone https://github.com/tssala23/ReadIssuesTest.git
+        cd ReadIssuesTest
+        
+    - name: create-patch
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        echo '$PAREMETER_FOR_THE_JSON_PAYLOAD' | yq e "" -P - > /tmp/data.yaml
+        mustache /tmp/data.yaml /mnt/config/robozome/onboarding.patch > /tmp/onboarding.patch
+        
+    - name: apply-push-patch
+      image: opf-toolbox
+      script:
+        #!/usr/bin/bash
+        
+        NAMESPACE=$(yq e ".namespace" /tmp/data.yaml)
+        CLUSTER=$(yq e ".cluster" /tmp/data.yaml)
+        GROUP=$(yq e ".group" /tmp/data.yaml)
+        DESCRIPTION=$(yq e ".description" /tmp/data.yaml)
+        USERS=$(yq e ".users" /tmp/data.yaml)
+        QUOTA=$(yq e ".quota" /tmp/data.yaml)
+        GPGKey=$(yq e ".gpg" /tmp/data.yaml)
+        
+        git checkout -b "onboarding/$NAMESPACE
+        git apply /tmp/onboarding.patch
+        git add ./cluster-scope/base/core/namespaces/$NAMESPACE/kustomization.yaml
+        git add ./cluster-scope/base/core/namespaces/$NAMESPACE/namespace.yaml
+        git add ./cluster-scope/overlays/prod/$CLUSTER/kustomization.yaml
```

Comment:
Is there a reason why to specifically select which files to stage for commit? Why don't you just `git add .`

Reply:
The advantage to using `git add .` is that you won't need to remember to update this script when the patch is modified to create different/additional files.